wire-library/wire-runtime-swift/src/main/swift/propertyWrappers/StringEncoded+Data.swift

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2023 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+extension Data : StringCodable {
```

Comment:
The methods in this extension could be `internal`. Maybe that'll change if we get rid of the `StringEncoded` property wrapper, but feels like we should start with them being internal and expose them later if needed.

Reply:
In either world, I need `StringCodable` to exist, and thus these methods need to be `public`.